clang/test/CodeGenSYCL/offload-fp32-div-sqrt.cpp

```diff
@@ -339,6 +357,15 @@ int main() {
     });
   });
 
+  deviceQueue.submit([&](handler& cgh) {
+    cgh.parallel_for<class KernelHalfSqrt>(numOfItems,
+    [=](id<1> wiID) {
+      // NFPA: call spir_func float @fdiv(ptr noundef byval({{.*}}) align 1 {{.*}}, ptr noundef byval({{.*}}) align 1 {{.*}})
+      // NFPA-FAST: call reassoc nnan ninf nsz arcp afn spir_func nofpclass(nan inf) float @fdiv(ptr noundef byval({{.*}}) align 1 {{.*}}, ptr noundef byval({{.*}}) align 1 {{.*}})
+      a[0] = fdiv(HalfValue1, HalfValue2);
```

Comment:
I'm not sure, if it is a real-world example, do e have fdiv builtin in SYCL/OpenCL/C++?. What about `a[0] = Value1 / Value2;`, can the issue be observed in this case and does your patch fix it?

Reply:
@MrSidims Thanks for the review. Fixed it.